test/unittests/test_parameters.py

```diff
@@ -228,6 +228,10 @@ def test_diff(self):
         self.assertEqual(P1.diff(P3),
                          ({'y': 3}, {'y': 4, 'z': 4}))
 
+    def test_long_parameter(self):
+        parameters = "x = '{}'".format(''.join("a" * 250))
```

Comment:
Great, thanks for changing this! The only thing that's failing now is the Travis build for Python 2.6 because that old version of Python requires explicit indices in the formatting spec (which continually trips me up on Travis, too). You can fix it by writing `"x = '{0}'".format(...)` instead of `"x = '{}'".format(...)`).